modules/Search/components/Facet.module.scss

```diff
@@ -1,27 +1,53 @@
 .container {
-    padding: $spacing-4;
+    @include not-desktop {
+        width: 100%;
+    }
+
+    @include tablet-up {
+        &:not(:last-child) {
+            margin-right: $spacing-3;
+        }
+    }
 
-    &:not(:last-child) {
-        border-bottom: 1px solid $color-borders;
+    @include mobile-only {
+        &:not(:last-child) {
+            margin-bottom: $spacing-1;
+        }
+    }
+
+    .button {
+        @include border-radius-m;
+
+        background: $color-base-white;
+        border: 1px solid $color-borders;
+        color: $color-base-600;
+        width: 100%;
+
+        &:hover {
+            opacity: 1;
+            color: $color-base-700;
+        }
+
+        &:active {
+            color: $color-base-700;
+        }
+    }
+
+    .buttonContent {
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+        width: 100%;
     }
 }
 
-.header {
-    @include text-label;
+.menu {
```

Comment:
❌ 
I find the horizontal spacing on the dropdown too narrow.
<img width="528" alt="image" src="https://user-images.githubusercontent.com/1641218/180998949-a6f9e1d9-2cbe-454a-8b4b-b7ba6a330a6c.png">

Especially compared to the header dropdowns:
<img width="234" alt="image" src="https://user-images.githubusercontent.com/1641218/180999041-d25e6ff9-a728-4809-a8f4-87538f4039cd.png">

There are also no interaction styling for the Facet items (even a simple pointer cursor).

Reply:
There seems to be space missing around the `Filters` button on mobile.
<img width="451" alt="image" src="https://user-images.githubusercontent.com/1641218/180999375-d0084f84-0b38-4434-a463-7b75df56c783.png">

Reply:
Making the update